books/std/util/support.lisp

```diff
@@ -390,7 +390,15 @@ saying whether the key was bound.  Returns (mv value boundp)."
           (mv nil (cdr x)))
          ((mv pre post)
           (split-/// ctx (cdr x))))
-      (mv (cons (car x) pre) post))))
+      (mv (cons (car x) pre) post)))
+
+  (defthmd true-listp-of-split-///
+    (b* (((mv pre-/// post-///) (std::split-/// ctx x)))
+      (implies (true-listp x)
```

Comment:
I think the hyp could apply only to the conclusion about post -- pre should be a true-list regardless